packages/docs/docs/contributing/troubleshooting.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Troubleshooting
+---
+
+This guide helps you resolve common issues when developing for Actual.
+
+## Type Errors
+
+### Issue: TypeScript compilation errors
+
+**Solution:**
+
+1. Run `yarn typecheck` to see all type errors
+2. Check if types are imported correctly
+3. Look for existing type definitions in `packages/loot-core/src/types/`
+4. Use `satisfies` instead of `as` for type narrowing
+
+### Issue: Cannot find module or type definitions
+
+**Solution:**
+
+1. Check `tsconfig.json` for path mappings
+2. Verify platform-specific imports (`.web`, `.electron`, `.api`)
+3. Ensure you're using the correct import path for the package
+4. Run `yarn install` to ensure all dependencies are installed
+
+## Linter Errors
+
+### Issue: ESLint or Prettier errors
+
+**Solution:**
+
+1. Run `yarn lint:fix` to auto-fix many issues
+2. Check ESLint output for specific rule violations
+3. Common custom rules:
+   - `actual/no-untranslated-strings` - Add i18n translation
+   - `actual/prefer-trans-over-t` - Use Trans component instead of t() function
+   - `actual/prefer-logger-over-console` - Use logger instead of console
+4. Check `eslint.config.mjs` for complete rules list
+
+### Issue: Import order errors
+
+**Solution:**
+
+ESLint automatically organizes imports. The order should be:
+
+1. React imports (first)
+2. Built-in Node.js modules
+3. External packages
+4. Actual packages
+5. Parent imports
+6. Sibling imports
+7. Index imports
+
+Run `yarn lint:fix` to automatically fix import order.
```

Comment:
This isn't a "common issue" if eslint:fix is used, which we've mentioned a lot in the other docs. It will also be autofixed in PR